container-search/src/test/java/com/yahoo/prelude/fastsearch/test/FastSearcherTestCase.java

```diff
@@ -321,7 +338,6 @@ public void testThatPropertiesAreReencoded() throws Exception {
         answer.flip();
         answer.get(expected);
 
-        assertEquals(expected.length, actual.length);
```

Comment:
Why not verify that length is the same. If actual is longer than expected, I assume that is incorrect.

Reply:
It's redundant; The method which checks array equality starts by checking length.